apps/alpha-test-app/src/components/layout-demo/LayoutSettings.vue

```diff
@@ -0,0 +1,165 @@
+<script lang="ts" setup>
+import { computed, ref } from "vue";
+import SingleSettingsGroup from "./SingleSettingsGroup.vue";
+import MultiSettingsGroup from "./MultiSettingsGroup.vue";
+import { watch } from "vue";
+import type { SelectionOption } from "sit-onyx";
+import { OnyxHeadline } from "sit-onyx";
+
+export type Settings = {
+  longPageContent?: boolean;
+  showFlyout?: boolean;
+  forceTooltip?: boolean;
+  showStickyContent?: boolean;
+  showToast?: boolean;
+
+  showSideBar?: boolean;
+  showSideBarCollapse?: boolean;
+  showTempOverlayTransparent?: boolean;
+  showTempOverlay?: boolean;
+
+  detailFooter?: boolean;
+  fullFooter?: boolean;
+
+  showPopover?: boolean;
+  showMobileFlyIn?: boolean;
+  showPageLoader?: boolean;
+  topBarFlyout?: boolean;
+};
```

Comment:
```suggestion
export type Settings = Partial<{
  longPageContent: boolean;
  showFlyout: boolean;
  forceTooltip: boolean;
  showStickyContent: boolean;
  showToast: boolean;

  showSideBar: boolean;
  showSideBarCollapse: boolean;
  showTempOverlayTransparent: boolean;
  showTempOverlay: boolean;

  detailFooter: boolean;
  fullFooter: boolean;

  showPopover: boolean;
  showMobileFlyIn: boolean;
  showPageLoader: boolean;
  topBarFlyout: boolean;
}>;
```

(optional) we can wrap the type in `Partial` so we don't need `?` for every key

Reply:
Is there a reason why some keys start with "show" and others don't?

Reply:
good point, I'll adjust :)

Reply:
I left the forceTooltip without "show" because it will always show on hover :) the force will just overrule the hover behavior :)